### PR TITLE
Reader: Remove fade from site description

### PR DESCRIPTION
**Pull request by blowery**
before
screenshot 2017-05-23 16 09 07

after
screenshot 2017-05-23 16 08 46

### CODE REVIEW

File: client/blocks/reader-feed-header/style.scss [hunk 1]
```diff
@@ -123,11 +123,7 @@
 
 		.reader-feed-header__description {
 
-				&::after {
-					@include long-content-fade( $size: 15% );
-					height: 16px * 1.3;
-					top: auto;
-			}
```
Comment (jancavan) [inline client/blocks/reader-feed-header/style.scss]: Empty block?

**Comment by jancavan** [comment]
I think it's uncommon to have extremely long descriptions so we can remove the limit instead of the fade getting added when unnecessary. 

I did set the a limit for `<660px` so content doesn't pushed way down for sites with longer descriptions: https://github.com/Automattic/wp-calypso/pull/14392/commits/589840959555830a2e75a161e8e14aed362edae6

**Comment by blowery** [comment]
nice, thanks @jancavan !

**Comment by bluefuton** [comment]
Sidenote: are we fading the title slightly too early? (Shouldn't hold up this PR though.)

screen shot 2017-05-24 at 09 54 12

**Comment by blowery** [comment]
> Sidenote: are we fading the title slightly too early? The whole title fit on screen here.

Yeah, this is why I dislike the fade. An ellipsis done by the browser would handle this properly and consistently.

**Comment by jancavan** [comment]
Yeah, it isn't very flexible. I think the fade was created because of multi-line text issues with the ellipsis.

**Pull request merged**